.env.example

```diff
@@ -1,9 +1,11 @@
-## Docker settings
+# This composition of compose files reflects the old one
+COMPOSE_FILE=docker-compose.yaml:distrib/compose/docker-compose.traefik.yaml:distrib/compose/docker-compose.traefik-labels.yaml:distrib/compose/docker-compose.mail-test.yaml
+# # Docker settings
```

Comment:
minor: add a new line before `# # Docker settings`

Reply:
No. That was the first line it's gone, it's pointless to start with an empty line.